cmd/svcat/class/get_cmd.go

```diff
@@ -81,7 +87,7 @@ func (c *getCmd) getAll() error {
 		return err
 	}
 
-	output.WriteClassList(c.Output, classes...)
+	output.WriteClassList(c.Output, c.outputFormat, classes...)
```

Comment:
This is my mistake. `c.App.RetrieveClasses()` should return the entire type and not just items, like bindings does.

What do you think?

Reply:
you go it!